implementations/rust/ockam/ockam_entity/src/credential/traits.rs

```diff
@@ -159,6 +159,8 @@ pub struct Credential {
     type_id: String,
 }
 
+impl ockam_core::Message for Credential {}
+
```

Comment:
This should probably use the `#[derive(Message)]` macro

Reply:
For a while during this PR I was bit by https://github.com/rust-analyzer/rust-analyzer/issues/10540, which made me think that wasn't working for some reason.